GliaWidgets/Sources/Screensharing/ScreenShareHandler.Live.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+extension ScreenShareHandler {
+    static let live: ScreenShareHandler = {
+        let status = ObservableValue<ScreenSharingStatus>(with: .stopped)
+        var visitorState: CoreSdkClient.VisitorScreenSharingState?
+
+        return ScreenShareHandler(
+            status: { status },
+            updateState: { state in
+                visitorState = state
+                guard let state = state else {
+                    status.value = .stopped
+                    return
+                }
+                switch state.status {
+                case .sharing:
+                    status.value = .started
+                case .notSharing:
+                    status.value = .stopped
+                @unknown default:
+                    break
+                }
+            },
+            stop: { completion in
+                visitorState?.localScreen?.stopSharing()
```

Comment:
This part contains some important business logic. like  calling some methods and `nil`-ing out state etc. Taking into account that we do not use `live` implementation it unit tests we will miss out opportunity to write tests for this logic. For me it seems like this part is totally testable as is, so maybe it should not be called live. We can discuss it separately.

Reply:
- added tests for `ScreenShareHandler`
- renamed Live with Implementation
- added `create()` static method
- replaced `.live` instance in tests with `.mock`
c17663b6a91b022e075a6c811e0fefa5a9ae7420